docs/src/submodule.md

```diff
@@ -198,5 +198,5 @@ Domain: Submodule over Integers with 1 generator and no relations
 Codomain: Free module of rank 2 over Integers)
 
 julia> I = intersect(N1, N2)
-Union{AbstractAlgebra.Generic.ModuleHomomorphism{BigInt}, AbstractAlgebra.Generic.Submodule{BigInt}}[]
+Any[]
```

Comment:
Oh wait, I've changed it here. But why? I think this happened as part of running `doctest(:fix)` or something like that... ?!? Huh. Anyway, if that explains it, then I am happy, phew :-)

Reply:
Maybe you ran `:fix` on a different version?